bookwyrm/templates/rss/edition.html

```diff
@@ -0,0 +1,8 @@
+{% blocktrans %}{{{obj.title}} by {{obj.author_text}}{% endblocktrans %}
```

Comment:
My bad, I didn't quite understand how _blocktrans_ works:

```suggestion
{% load i18n %}
{% blocktrans trimmed with book_title=obj.title book_author=obj.author_text %}
  ‘{{ book_title }}’ by {{ book_author }}
{% endblocktrans %}
```